### DISCUSSION TITLE
Where do I submit the Pre-Proposal

### DISCUSSION THREAD

**Discussion by gluonparticle** [MDAnalysis/mdanalysis#4546] [GSoC Q&A] [answered]
I am really excited about the opportunity to work with MD Analysis team.

I shared my [idea for the GSoC project](https://github.com/MDAnalysis/mdanalysis/discussions/4525) in GitHub discussions and I haven't had any response from the team. Is GitHub the correct platform for the pre-proposal or should I do the same on Discord? 

I am sorry but I cannot find the Discord invite link to join the server despite my best attempts. The link in GSoC page contains the link to the channel but not to join the server.

**Comment by orbeckst** [comment, marked as answer]
The invite link for discord can be found at https://www.mdanalysis.org/#participating

The [deadline for the pre-proposal](https://www.mdanalysis.org/2024/02/27/gsoc2024/#submit-a-pre-proposal) was **March 15** so I think you're too late to participate.
